src/main/kotlin/no/skatteetaten/aurora/boober/controller/v1/InformationControllerV1.kt

```diff
@@ -0,0 +1,13 @@
+package no.skatteetaten.aurora.boober.controller.v1
+
+import no.skatteetaten.aurora.boober.controller.internal.Response
+import no.skatteetaten.aurora.boober.service.BitbucketService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class InformationControllerV1(val bitbucketService: BitbucketService) {
+
+    @GetMapping("/v1/auroraconfig")
+    fun auroraConfigs(): Response = Response(items = bitbucketService.auroraConfigs())
```

Comment:
Siden vi ikke har en AuroraConfigService (som jeg håper vi klarer få etterhvert), syntes jeg dette bør gå via DeploymentBundleService. F.eks. findAllAuroraConfigNames().

Reply:
Når vi enabler at brukere kan pushe direkte til AC repoene må vi sjekke om man har lov til dette før man gjør en edit/deploy. Da trenger vi en metode som sjekker om en ident har lov til å skrive til en slug i et prosjekt. Derfor laget jeg en helt ny Service for å abstrahere rundt alle operasjoner som må gjøres mot bitbucket.